nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -424,6 +424,37 @@
      installer after creating <literal>/var/lib/nextcloud</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The <literal>/etc/sysctl.d/nixos.conf</literal> file containing all the options set via
+      <link linkend="opt-boot.kernel.sysctl">boot.kernel.sysctl</link> was moved to
+      <literal>/etc/sysctl.d/60-nixos.conf</literal>, as <literal>sysctl.d(5)</literal>
+      recommends prefixing all filenames in <literal>/etc/sysctl.d</literal> with a
+      two-digit number and a dash to simplify the ordering of the files.
+    </para>
+   </listitem>
+   <listitem>
+     <para>
+       We now install the sysctl snippets shipped with systemd. Apart from enabling loose reverse path
+       filtering and source routing filtering, this enables <literal>fq_codel</literal> as a packet
+       scheduler to fight bufferbloat, and configures the kernel to pass coredumps to
+       <literal>systemd-coredump</literal>.
+       These sysctl snippets can be found in <literal>/etc/sysctl.d/50-*.conf</literal>,
+       and overridden via <link linkend="opt-boot.kernel.sysctl">boot.kernel.sysctl</link>
+       (which will place the parameters in <literal>/etc/sysctl.d/60-nixos.conf</literal>).
+     </para>
+   </listitem>
+   <listitem>
+    <para>
+      The <literal>systemd.coredump</literal> module has been merged into the systemd module.
```

Comment:
I don't think the code changes need to be in the release notes, any information that would be important for end users.

Reply:
Right - I dropped that line.